library/src/main/kotlin/org/codice/compliance/Common.kt

```diff
@@ -73,6 +79,21 @@ class Common {
                     ?.first { it.binding == binding }
                     ?.location
         }
+
+        fun prettyPrintXml(xml: String): String {
```

Comment:
✏️ This is a pretty specific function (with a nicely specific/clear name) to be performed on a String; it is a good candidate for an extension function.